IntelliTect.Analyzer/IntelliTect.Analyzer/DiagnosticUrlBuilder.cs

```diff
@@ -1,23 +1,32 @@
 using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
 
 namespace IntelliTect.Analyzer
 {
 #pragma warning disable CA1055 // Uri return values should not be strings
-    
+
     public static class DiagnosticUrlBuilder
     {
-        private const string BaseUrl = "https://github.com/IntelliTect/CodingStandards/wiki/";
+        private const string BaseUrl = "https://github.com/IntelliTect/CodingGuidelines";
 
         /// <summary>
         /// Get the full diagnostic help url
         /// </summary>
-        /// <param name="analyzerBlock">Current analyzer block</param>
-        /// <param name="intlCode">The intl error code</param>
+        /// <param name="title">intl analyzer title</param>
```

Comment:
For comments we should avoid abbreviations.
```suggestion
        /// <param name="title">IntelliTect analyzer title</param>
```